managing-mods/git.md

```diff
@@ -0,0 +1,23 @@
+# Managing and Sharing Mods with Git
```

Comment:
Not in this PR pls. You can take it out to a separate one (but I’ll have some notes about how to take it as a whole, so the current content might end up being removed completely)

Reply:
You want to do git? Could you open PR or an issue and assign it to yourself?
Then I'll scrap this git.md

Reply:
Probably yes (though I have no idea when I’ll be able to write it)